index.d.ts

```diff
@@ -118,6 +118,15 @@ declare namespace electronDl {
 		@default false
 		*/
 		readonly overwrite?: boolean;
+
+		/**
+		Save dialog options used to personalize the dialog.
+
```

Comment:
It should mention that it set the `defaultPath` for the user if not explicitly defined.

Reply:
Hope the amended description is OK, let me know if further changes are required